cluster/addons/dns/kube-dns.yaml.base

```diff
@@ -83,7 +83,38 @@ spec:
         k8s-app: kube-dns
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
+        # For 1.6, we keep the old tolerations in case of a downgrade to 1.5
```

Comment:
Could we get rid of these annotations? https://github.com/kubernetes/kubernetes/pull/52193#discussion_r139030187

Reply:
So delete lines 86 -107 here and the equivalents?

Reply:
Yep, that sounds correct to me.

Reply:
Are you asking me? I have no way of telling if it is at all common to have the newer kube-dns used by older kubernetes versions, so I have no opinion.

Reply:
IMHO we shouldn't be carrying the deprecated alpha annotations on the upstream kube-dns manifest. Specific k8s deployment could have its own addon configuration to fit its use case, but that should be maintained separately. So I'd prefer we delete them here.